### PR TITLE
Weekly Changelog 2024-11-18 to 2024-11-25

### PR DESCRIPTION
**Pull request by kevingranade**
#### Summary
None

#### Additional context
`Automatically generated as a draft. Please copy-edit before merging.`

### CODE REVIEW

**Comment by harakka** [comment]
@Night-Pryanik please give these PRs more time in draft so that people have time to copyedit them.

**Comment by Night-Pryanik** [comment]
Ok.

File: data/changelog.txt [hunk 1]
```diff
@@ -511,6 +521,11 @@ Crafting GUI Search: cancel the search with ESC
 use the monospaced font for the ascii art on the end screen
 Increased available space for skills chapter in description tab of character creation menu
 ImGui inherits theme colors
+Sidebar widget to show faction territory
+Revert horizontal separator color back to dark gray
```
Comment (Maleclypse) [inline data/changelog.txt:525]: @Night-Pryanik hey Boss, these really need to be copy-edited by someone before merge. This is the abbreviated changelog that should only contain notable changes. This revert for instance shouldn’t have been included.

File: data/changelog.txt [hunk 2]
```diff
@@ -833,6 +848,18 @@ Make the cryptid mission assigning less likely to hang
 Aftershock: Disable portal storms in Exoplanet
 Martial Mastery: Add three new defensive perks.
 [Magiclysm] Fix demonic possession causing starvation penalties
+[Xedra Evolved] More Ierde powers
```
Comment (Maleclypse) [inline data/changelog.txt:851]: Neither should this one

File: data/changelog.txt [hunk 3]
```diff
@@ -833,6 +848,18 @@ Make the cryptid mission assigning less likely to hang
...
+Add more Lilin traits and features
```
Comment (Maleclypse) [inline data/changelog.txt:852]: Nor this one

File: data/changelog.txt [hunk 4]
```diff
@@ -833,6 +848,18 @@ Make the cryptid mission assigning less likely to hang
...
+Reviewed Magiclysm Monsters and added proficiency families and weakpoints when missing.
+Allow Lilin to drain ruach while under the effects of NO_SPELLCASTING
+[MoM] Add the PATIENT MOSAIC research room to skyscraper lab
+[MoM] Parameterize PHAVIAN skyscraper lab
+[MoM] Add the Gamma Sterilization photokinetic power
+[Magiclysm] Fix soul afterburner granting infinite moves
```
Comment (Maleclypse) [inline data/changelog.txt:858]: This should’ve been in bugfixes or not included

File: data/changelog.txt [hunk 5]
```diff
@@ -833,6 +848,18 @@ Make the cryptid mission assigning less likely to hang
...
+[MoM] Update creature mindsight to be variable, apply to Eater
+[Xedra Evolved] Add Lilit ruach gauge using custom variable functionality
+[Magiclysm] Add the mana-shackled challenge trait
+[Xedra Evolved] Arvore Cleanup
```
Comment (Maleclypse) [inline data/changelog.txt:862]: Definitely not this one

File: data/changelog.txt [hunk 6]
```diff
@@ -950,6 +977,15 @@ Nerf atlatls, a lot.
 Martial arts manuals spawn in libraries and houses
 Lower Stream Item Spawns
 Audit Lead Acid Batteries breaks_into
+Add missing longest_side
```
Comment (Maleclypse) [inline data/changelog.txt:980]: Too vague to be included

File: data/changelog.txt [hunk 7]
```diff
@@ -950,6 +977,15 @@ Nerf atlatls, a lot.
...
+Mining solid rock now results in a lot more rock and very rarely, you can find a bit of limestone while digging in soil
+No Hope: Reduce spawn chance of broken cyborgs in bionic basements
+Food price scales with freshness
+Aftershock: Reduce spawn chance of broken cyborgs in bionic basements
```
Comment (Maleclypse) [inline data/changelog.txt:984]: Mods or not mentioned at all

File: data/changelog.txt [hunk 8]
```diff
@@ -950,6 +977,15 @@ Nerf atlatls, a lot.
...
+Dry stone walls now need to be climbed over
+Change spear techniques from impaling strike to penetrating strike
+Changed Survivor Hood Materials and Added Filtration to Cowls
```
Comment (Maleclypse) [inline data/changelog.txt:987]: Removed

File: data/changelog.txt [hunk 9]
```diff
@@ -1288,6 +1324,28 @@ Fix duplicate an in exterminatron description
 [Xedra Evolved] Add dinomod check for lilit kcal drain
 remove debug message in add_profession_items()
 Fix nunez bullet trailer appearing out of nowhere
+[Xedra Evolved] Gracken can't learn dream magick either
+Enable Other Anvils for Isherwood Quest
+Fix mining using a pickaxe always causes an error
+Rural church no longer spawns water heaters in walls
+Resolve Milling Issues/Errors
+NECC mapgen wrong symbol fix
```
Comment (Maleclypse) [inline data/changelog.txt:1332]: Remove

File: data/changelog.txt [hunk 10]
```diff
@@ -1288,6 +1324,28 @@ Fix duplicate an in exterminatron description
...
+Evac shelter no longer has dirt walls
+Fix camp expansion recipes not being available
+drying_rate test displays weather, clears map
+Decreased volume of pillows but made them a little bit heavier
```
Comment (Maleclypse) [inline data/changelog.txt:1336]: Remove

File: data/changelog.txt [hunk 11]
```diff
@@ -1288,6 +1324,28 @@ Fix duplicate an in exterminatron description
...
+fix archery_damage_test CI occasional failure
+Fix `bio_blindfold` not working correctly after a save & load cycle
+Make wood axe usable for great axe proficiency practice instead of hand axe proficiency practice
```
Comment (Maleclypse) [inline data/changelog.txt:1339]: I wouldn’t have left this in but others could make different choices

File: data/changelog.txt [hunk 12]
```diff
@@ -1288,6 +1324,28 @@ Fix duplicate an in exterminatron description
...
+Don't run item::on_pickup when the item is wielded
+Autodrive doesn't stop due to driving over sludge etc
+better margin of error for test shot_custom_damage_type
+Fix format mismatches in tripoint item vars
+Fix crashing in CURSES without ASCII title art
+Fix penetrating strike
+ALLOWS_TALONS is about talons for feet, not for hands
+Fix headlamp to accept medium batteries akin to a flashlight
```
Comment (Maleclypse) [inline data/changelog.txt:1347]: Too minor

File: data/changelog.txt [hunk 13]
```diff
@@ -1425,6 +1485,12 @@ Roof palette has more built-in nests
 obsolete unused generic NPC classes
 abstractify basic walls
 u_/npc_forget_recipe can use categories and subcategories in place of individual ids
+Add std::string wrappers for ImGui::InputText* methods
+Adds a guide to the development process
+Allow variable objects and math expressions to be used as widget variables
+fill_ter accepts parameters
+Merge commits from 0-H-Branch into master and update appstream file
```
Comment (Maleclypse) [inline data/changelog.txt:1492]: Definitely not

File: data/changelog.txt [hunk 14]
```diff
@@ -1472,6 +1540,9 @@ Obsolete strings now have a comment with an explanation
 Make Portugal Portugues appear in Options
 [AFS] Add NO_I18N/PREFIX_XL
 [Magiclysm] NO_I18N, dont extract monster/fake spells
+[Xedra Evolved] NO_I18N, dont extract monster/fake spells
```
Comment (Maleclypse) [inline data/changelog.txt:1543]: Combine this and the line above

File: data/changelog.txt [hunk 15]
```diff
@@ -1472,6 +1540,9 @@ Obsolete strings now have a comment with an explanation
...
+Restructure the template file
+[MoM] Dont extract monster/fake spells/effects for translation
```
Comment (Maleclypse) [inline data/changelog.txt:1545]: Combine this with the other two identical PRs

**Review by Maleclypse — COMMENTED** [review]
If you want to merge changelog PRs please edit them. If you merge them without editing someone has to go back and do them all again in the future at the next stable candidate.

**Comment by Night-Pryanik** [comment]
Ok, I won't be touching these PRs ever again.

**Comment by Maleclypse** [comment]
> Ok, I won't be touching these PRs ever again.

That's fair if you choose to do so. I'm not asking you not to do it. I'm asking that you follow the instructions in the PR. Either decision is fine.

**Comment by Maleclypse** [comment]
> Ok, I won't be touching these PRs ever again.

I apologize that you got two comments on this. I’m sure that seemed like dog piling. I had opened s review to give you an idea of how best to help on this and I guess harraka commented while I was making the review. 

Absolutely not the intention to drag you with a bunch of comments.

**Pull request merged**